client/export_test.go

```diff
@@ -30,9 +30,11 @@ func (client *Client) SetDoer(d doer) {
 	client.doer = d
 }
 
+type DoFlags doFlags
```

Comment:
no, you want a type alias here, not a new type. I.e. you want
```suggestion
type DoFlags = doFlags
```